syntax = "proto3";

package gkesite;

service HelloService {
    rpc GetGreeting(GreetingRequest) returns (Greeting) {}
}

message GreetingRequest {
    string name = 1;
}

message Greeting {
    string message = 1;
}

service SquareService {
    rpc GetSquare(SquareRequest) returns (SquareResponse) {}
}

message SquareRequest {
    float number = 1;
}

message SquareResponse {
    float number = 1;
}

service CountService {
    rpc GetCount (CountRequest) returns (CountResponse) {}
}

message CountRequest {
}

message CountResponse {
    int32 count = 1;
}
